airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/ssh/SshTunnel.java

```diff
@@ -231,13 +229,12 @@ public void close() {
    * From the RSA format private key string, use bouncycastle to deserialize the key pair, reconstruct
    * the keys from the key info, and return the key pair for use in authentication.
    */
-  private KeyPair getPrivateKeyPair() throws IOException {
-    final PEMParser pemParser = new PEMParser(new StringReader(validateKey()));
-    final PEMKeyPair keypair = (PEMKeyPair) pemParser.readObject();
-    final JcaPEMKeyConverter converter = new JcaPEMKeyConverter();
-    return new KeyPair(
-        converter.getPublicKey(SubjectPublicKeyInfo.getInstance(keypair.getPublicKeyInfo())),
-        converter.getPrivateKey(keypair.getPrivateKeyInfo()));
+  KeyPair getPrivateKeyPair() throws IOException, GeneralSecurityException {
```

Comment:
Is there a reason this is no longer `private` and if so how come? 

Minor nit: not a fan of using `kp` as a variable name since it's not abundantly clear what that means. I presume it means `keypair`

Reply:
This is required for testing. Reduced visibility to package private, and changed name.
thanks!